### PR TITLE
PR-#Resetting all settings from within the UI menu.

### PR DESCRIPTION
**Pull request by GeminiServer**
Able to reset the customer setting from the menu. It will bring a waring dialog to ensure to reset by mistake.  So no need to add a file to sd-card to reset the settings. Ignore lcd mode only if you reset all settings and to show the info dialog. Otherwise, it won't show if you currently in CLEAN_MODE_SWITCHING_SUPPORT.

### CODE REVIEW

**Comment by thisiskeithb** [comment]
> So no need to add a file to sd-card to reset the settings

This only works if you can access the UI. If the touch calibration is wrong, the SD card reset method is the only way to reset it.

File: TFT/src/User/Menu/FeatureSettings.c [hunk 1]
```diff
@@ -43,6 +43,9 @@ const LABEL itemMoveSpeed[ITEM_SPEED_NUM] = {
                                             };
 const  u8 item_movespeed[ITEM_SPEED_NUM] = {LABEL_NORMAL_SPEED, LABEL_SLOW_SPEED, LABEL_FAST_SPEED};
 
+#define ACTION_NUM   1
+const LABEL itemAction[ACTION_NUM]       = {LABEL_RESET};
+
```
Comment (guruathwal) [inline TFT/src/User/Menu/FeatureSettings.c]: These lines are not required as the Custom Value of the `Reset settings` item does not changes.
```suggestion
```

File: TFT/src/User/Menu/FeatureSettings.c [hunk 2]
```diff
@@ -208,6 +236,12 @@ void updateFeatureSettings(uint8_t key_val)
     break;
     #endif
 
+    case SKEY_RESET_SETTINGS:
+    settingPage[item_index].valueLabel = itemAction[0];
```
Comment (guruathwal) [inline TFT/src/User/Menu/FeatureSettings.c]:
```suggestion

```

File: TFT/src/User/Menu/FeatureSettings.c [hunk 3]
```diff
@@ -280,12 +314,15 @@ void loadFeatureSettings(){
         settingPage[item_index].icon  = toggleitem[infoSettings.file_listmode];
         featureSettingsItems.items[i] = settingPage[item_index];
         break;
-
+      case SKEY_RESET_SETTINGS:
+        settingPage[item_index].valueLabel = itemAction[0];
```
Comment (guruathwal) [inline TFT/src/User/Menu/FeatureSettings.c]:
```suggestion

```

**Pull request merged**